### PR TITLE
Test consistency on cloud-provider-config

### PR DESCRIPTION
**Pull request by rlobillo**
test_coverage for https://bugzilla.redhat.com/show_bug.cgi?id=2065597

### CODE REVIEW

**Review by mandre — COMMENTED** [review]
Thanks @rlobillo.
I see we're checking the different config maps exist and contain expected entries.
Should we also check that these config maps are updated to the new expected value if we modify the user managed cloud.conf?

For example, changing a param in the `[BlockStorage]` section should update the `openshift-config-managed/kube-cloud-config` and `openshift-cluster-csi-drivers/cloud-conf` config maps but not `openshift-cloud-controller-manager/cloud-conf`.

**Comment by mandre** [comment]
> > Thanks @rlobillo. I see we're checking the different config maps exist and contain expected entries. Should we also check that these config maps are updated to the new expected value if we modify the user managed cloud.conf?
> > For example, changing a param in the `[BlockStorage]` section should update the `openshift-config-managed/kube-cloud-config` and `openshift-cluster-csi-drivers/cloud-conf` config maps but not `openshift-cloud-controller-manager/cloud-conf`.
> 
> AFAIK, changing a param will imply a reconfiguration on the cluster. For example, in the case of changes in the '[LoadBalancer]' section, it will imply a sequential reboot of all the nodes. We are doing that on D/S CI and it is taking ~30 minutes. ([Reference](http://git.app.eng.bos.redhat.com/git/openshift-ir-plugin.git/tree/roles/tests/tasks/change_cloud_cm.yml))

Fair enough, the sync of the config maps should be quick, but I forgot this will trigger the kubelet to restart one by one leading to cluster instability. Forget about my ask then.

File: test/extended/openstack/cloud-provider-config.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+		})
+
+		// https://bugzilla.redhat.com/show_bug.cgi?id=2065597
+		g.It("should haul the user config to the expected config maps", func() {
+
+			type config_params struct {
+				namespace string
+				name      string
+				key       string
+				skip      []string //Slice of properties that are not expected to be present.
```
Comment (MaysaMacedo) [inline test/extended/openstack/cloud-provider-config.go:38]:
```suggestion
				skip      []string //Slice of properties that are not expected to be present in the internal configMaps
```

Comment (rlobillo) [reply test/extended/openstack/cloud-provider-config.go]: Thanks, done.

**Pull request merged**